src/xml.tsx

```diff
@@ -101,31 +106,37 @@ export function SvgAst({ ast, override }: AstProps) {
   );
 }
 
+export const err = console.error.bind(console);
+
 export function SvgXml(props: XmlProps) {
-  const { xml, override } = props;
+  const { onError = err, xml, override } = props;
   const ast = useMemo<JsxAST | null>(() => (xml !== null ? parse(xml) : null), [
     xml,
   ]);
-  return <SvgAst ast={ast} override={override || props} />;
+
+  try {
+    return <SvgAst ast={ast} override={override || props} />;
+  } catch (error) {
+    onError(error);
+    return null;
+  }
```

Comment:
Perhaps it would be more suitable with an Error boundary here?

Reply:
@msand, sorry for not taking the time to improve the PR based on your comments. I was planning to do that, but I started some projects and didn't have the time to do it.